### PR TITLE
TableHeaderColumnProps.dataFormat 3rd parameter.

### PR DESCRIPTION
**Pull request by siavelis**
Added 3rd parameter in TableHeaderColumnProps.dataFormat field. One can provide additional data to the 'dataFormat' function using 'formatExtraData'.

### CODE REVIEW

**Comment by scharf** [comment]
👍

**Review by scharf — APPROVED** [review]

File: react-bootstrap-table/react-bootstrap-table.d.ts [hunk 1]
```diff
@@ -476,7 +476,7 @@ declare module "react-bootstrap-table" {
 		To customize the column. This callback function should return a String or a React Component.
 		In addition, this function taking two argument: cell and row.
 		*/
-		dataFormat?: (cell:any, row:any) => string | ReactElement<any>;
+		dataFormat?: (cell:any, row:any, formatExtraData:any) => string | ReactElement<any>;
```
Comment (scharf) [inline react-bootstrap-table/react-bootstrap-table.d.ts]: hmm, I guess the new parameter should be optional `formatExtraData?:any`

**Review by scharf — CHANGES_REQUESTED** [review]
I think the new parameter should be optional

**Comment by andy-ms** [comment]
Please make this for the `types-2.0` branch instead. We are closing all pull requests targeting `master`. See #13233.

File: react-bootstrap-table/react-bootstrap-table.d.ts (hunk 1 above)
Comment (siavelis) [inline react-bootstrap-table/react-bootstrap-table.d.ts]: Yes indeed, it is optional.

**Comment by siavelis** [comment]
ok

**Pull request closed without merging**